src/head.js

```diff
@@ -3,4 +3,5 @@
  *
  * head :: [a] -> a
  */
-export default ([x, ...xs]) => x
+// eslint-disable-next-line no-unused-vars
+export default ([x, ...xs]) => x;
```

Comment:
What we could actually do here (I didn't realise at the time of writing), is this: 

```suggestion
export default ([x, ]) => x;
```

That way we can avoid the `eslint-disable` line.

Reply:
That would actually trigger a different linting error: "Unexpected trailing comma.". Should I change and disable that error ? Or keep it as is ?

Reply:
Oh you're right of course, but we don't need the comma. 
This'll do the trick:

```js
export default ([x]) => x;
```